skycultures/arabic_arabian_peninsula/Transliteration.md

```diff
@@ -1,37 +0,0 @@
-| Arabic letter | Transliteration |Pronunciation|
```

Comment:
Not sure, is this meta information/documentation now lost? Even if not used in the program, this and similar files in other SCs could contain valuable information.

Reply:
Hmm, apparently yes. I'll try to recover it, but I guess I won't put this ability into the converter. Or shall I?

Reply:
Not needed for the converter, there are too many loose ends here, some possible READMEs or other background info files. Just that IMO such files, which were important in the eyes of contributors who provided them, should not get lost. Probably any such background info can remain in the developers' repo, and just be excluded from being packed into the builds. (Did we not discuss to add an optional subdir for that?) Of course, also no need for setting up translation when we stick to English in the first place.

Github seems to not like me reviewing all the files of this 'giga' branch in the browser. Need to install the desktop app for more...